feature/login/src/main/java/com/example/login/viewmodel/EmailAuthViewModel.kt

```
@@ -0,0 +1,166 @@
+                    _authState.value = EmailAuthState.SendSuccess("인증 코드 전송 성공")
+                    startTimer() // 성공 시 타이머 시작
+                } else {
+                    _authState.value = EmailAuthState.SendError(AuthErrorMessages.SERVER_ERROR)
+                }
+            } catch (e: HttpException) {
+                Log.e("EmailAuthVM", "HttpException in sendEmailCode", e)
+                _authState.value = when (e.code()) {
+                    409 -> EmailAuthState.SendError(AuthErrorMessages.EMAIL_ALREADY_EXISTS)
+                    else -> EmailAuthState.SendError(AuthErrorMessages.SERVER_ERROR)
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)